.github/workflows/deploy-preprod-q1.yml

```diff
@@ -13,19 +13,12 @@ jobs:
     name: Build, push and deploy to dev-fss Q1
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout code
-        uses: actions/checkout@v1
-      - name: Read/save maven cache
-        uses: actions/cache@v1
+      - uses: actions/checkout@v2.4.0
```

Comment:
Når vi går over til v2 bør vi bruke fetch-depth: 0 også for å unngå trøbbel med sonarcloud på PRer
https://github.com/navikt/familie-tilbake/blob/8f91b5da3405c0d50fc07bbfaeed6c645a3dee0a/.github/workflows/build-and-deploy-dev.yml#L17

Reply:
Virker som at det går fint med 1, kan eventuellt endre senere